packages/cloud-cognitive/src/components/Tearsheet/_tearsheet.scss

```diff
@@ -1,219 +1,229 @@
 //
-// Copyright IBM Corp. 2020, 2020
+// Copyright IBM Corp. 2020, 2021
 //
 // This source code is licensed under the Apache-2.0 license found in the
 // LICENSE file in the root directory of this source tree.
 //
 
+@import '@carbon/import-once/scss/import-once';
 @import '../../global/styles/carbon-settings';
 @import '../../global/styles/project-settings';
 
 @import 'carbon-components/scss/components/modal/modal';
 
-$block-class: #{$pkg-prefix}-tearsheet;
-// stylelint-disable-next-line carbon/theme-token-use
-$overlay-color: $carbon--gray-100;
+// Define all component styles in a mixin which is then exported using
+// the Carbon import-once mechanism.
+@mixin example-component {
```

Comment:
This mixin should be named something like `@mixin tearsheet {}`, I'm guessing this was an accident.

Reply:
Oops! Good catch :-)

Reply:
Fixed.